### PR TITLE
Added a Note

### PR DESCRIPTION
**Pull request by joinimran**
A note is added that we need to enable PKU2U to RDP from hybrid Azure AD joined Server to Azure AD joined Windows 10 devices.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/5215

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 Please do a copy review.

File: windows/security/threat-protection/security-policy-settings/network-security-allow-pku2u-authentication-requests-to-this-computer-to-use-online-identities.md [hunk 1]
```diff
@@ -40,6 +41,9 @@ This policy is not configured by default on domain-joined devices. This would di
 
     This will allow authentication to successfully complete between the two (or more) computers that have established a peer relationship through the use on online IDs. The PKU2U SSP obtains a local certificate and exchanges the policy between the peer devices. When validated on the peer computer, the certificate within the metadata is sent to the logon peer for validation. It associates the user's certificate to a security token, and then the logon process completes.
 
+> [!Note]
+> KU2U is disabled by default on server SKUs and thus RDP from a hybrid Azure AD joined server to a Azure AD joined Windows 10 device or Hybrid Azure AD joined domain member Windows 10 device fails. To resolve this PKU2U needs to be enabled on server SKU.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/security-policy-settings/network-security-allow-pku2u-authentication-requests-to-this-computer-to-use-online-identities.md]:
```suggestion
> KU2U is disabled by default on server SKUs and thus RDP from a hybrid Azure AD joined server to a Azure AD joined Windows 10 device or Hybrid Azure AD joined domain member Windows 10 device fails. To resolve this, PKU2U needs to be enabled on the server SKU.
```

**Comment by joinimran** [comment]
Suggestions applied. Ready to go live.

**Comment by jvsam** [comment]
Thank you so much @joinimran!

**Comment by jvsam** [comment]
@Dansimp kindly review this PR and merge if you see no issues. Thank you so much.

@officedocsbot assign @Dansimp

**Comment by jvsam** [comment]
Hello again @Dansimp, just a friendly follow-up on the status of this PR. Thanks and regards!

**Comment by jvsam** [comment]
Hi @Dansimp, just following up on this PR. I hope you'll find time to review it. If there are issues/concerns, kindly let us know. Thank you.

File: windows/security/threat-protection/security-policy-settings/network-security-allow-pku2u-authentication-requests-to-this-computer-to-use-online-identities.md [hunk 2]
```diff
@@ -40,6 +41,9 @@ This policy is not configured by default on domain-joined devices. This would di
 
     This will allow authentication to successfully complete between the two (or more) computers that have established a peer relationship through the use on online IDs. The PKU2U SSP obtains a local certificate and exchanges the policy between the peer devices. When validated on the peer computer, the certificate within the metadata is sent to the logon peer for validation. It associates the user's certificate to a security token, and then the logon process completes.
 
+> [!Note]
+> KU2U is disabled by default on server SKUs and thus RDP from a hybrid Azure AD joined server to a Azure AD joined Windows 10 device or Hybrid Azure AD joined domain member Windows 10 device fails. To resolve this, PKU2U needs to be enabled on the server SKU.
```
Comment (illfated) [inline windows/security/threat-protection/security-policy-settings/network-security-allow-pku2u-authentication-requests-to-this-computer-to-use-online-identities.md]: minor grammatical details:
```suggestion
> KU2U is disabled by default on server SKUs and thus RDP from a hybrid Azure AD joined server to an Azure AD joined Windows 10 device, or Hybrid Azure AD joined domain member Windows 10 device, fails. To resolve this, PKU2U needs to be enabled on the server SKU.
```
(commas added to make it easier to get the overarching meaning of the sentence)

**Comment by joinimran** [comment]
Suggestion applied. Ready to go live. @Dansimp , over to you.

**Pull request merged**